linera-base/src/crypto/mod.rs

```diff
@@ -11,6 +11,7 @@ mod secp256k1;
 use std::{fmt::Display, io, num::ParseIntError, str::FromStr};
 
 use alloy_primitives::FixedBytes;
+use async_trait::async_trait;
```

Comment:
We support async traits natively in Rust now, so this mostly isn't needed any more.  Instead, try `trait-variant` for just the functionality of conditionally providing a `Send`/`?Send` definition of the trait.

Reply:
Done - although, force-pushed.